sale_line_returned_qty/tests/test_sale_line_returned_qty.py

```diff
@@ -57,9 +58,24 @@ def test_returned_qty(self):
         # Partial delivery one
         picking = self.order.picking_ids
         picking.action_assign()
-        picking.move_ids.quantity_done = 10.0
+        picking.move_ids.quantity = 10.0
+        picking.move_ids.picked = True
         picking._action_done()
+        self.assertEqual(picking.state, "done")
         self.assertEqual(so_line.qty_returned, 0.0)
         # Make a return for 5 units
-        self._return_picking(picking, 5.0, to_refund=True)
```

Comment:
Why not using the included helper function ?

Reply:
@rousseldenis This all requested changes are updated.